.travis-test.sh

```diff
@@ -15,5 +15,5 @@ echo "extension = ext/modules/maxminddb.so" >> ~/.phpenv/versions/"$(phpenv vers
 
 if [ $RUN_LINTER ]; then
     vendor/bin/php-cs-fixer fix --verbose --diff --dry-run --config=.php_cs
-    vendor/bin/phpcs --standard=.phpcs-ruleset.xml src/
+    vendor/bin/phpcs -n --standard=.phpcs-ruleset.xml src/
```

Comment:
We want the warnings. We don't want the long lines.

Reply:
This issue I explain on this [comment thread](https://github.com/maxmind/web-service-common-php/pull/25#discussion_r343407905).